bedrock/mozorg/views.py

```diff
@@ -122,23 +121,14 @@ def namespaces(request, namespace):
 
 @require_safe
 def home_view(request):
-    locale = l10n_utils.get_locale(request)
-
     variation = request.GET.get("v", None)
 
     if variation not in ["1", "2"]:
         variation = None
 
-    ctx = {"ftl_files": ["mozorg/home-new", "mozorg/home-mr2-promo"], "add_active_locales": ["de", "fr"], "variation": variation}
+    ctx = {"ftl_files": ["mozorg/home-new", "mozorg/home-mr2-promo"], "variation": variation}
 
-    if locale == "de":
-        template_name = "mozorg/home/home-de.html"
-        ctx["page_content_cards"] = get_page_content_cards("home-de", "de")
-    elif locale == "fr":
-        template_name = "mozorg/home/home-fr.html"
-        ctx["page_content_cards"] = get_page_content_cards("home-fr", "fr")
-    else:
-        template_name = "mozorg/home/home.html"
```

Comment:
**Note:** Removed this view because we're no longer going to have dedicated homepages for French and German locales. This homepage will be global.